HaulageApplication/Haulage/BaseClasses/Accounting/Administrator.cs

```diff
@@ -35,21 +35,82 @@ public void manageManifests()
 
         }
 
-        public void planRoutes()
+        public void AddManifestItem(TripManifest manifest, ManifestItem item)
```

Comment:
Some methods in the Administrator class use camelcase and some use pascalcase. Should this be made consistent?

Reply:
LJ - Updated now.